Dockerfile

```diff
@@ -1,29 +1,30 @@
-FROM golang:1.13 as secrets-provider-builder
+FROM golang:1.13 as secrets-provider-builder-base
 MAINTAINER CyberArk Software Ltd.
 
 ENV GOOS=linux \
     GOARCH=amd64 \
     CGO_ENABLED=0
 
-COPY . /opt/secrets-provider-for-k8s
+RUN apt-get update
```

Comment:
what's the purpose of this update?

Reply:
It was there, I just rearranged it and moved it to the top for improved performance.

Reply:
then you missed something. it was `RUN apt-get update && apt-get install -y jq`

Reply:
I can't see any reason why `jq` is needed for building Go.

Reply:
yes, it seems that it is not used. Anyway, in that case let's remove also the `apt-get update`.